docs/source/release_notes.rst

```diff
@@ -11,6 +11,7 @@ Future Release
         * Add ``RateOfChange`` primitive (:pr:`2359`)
     * Fixes
         * Sets ``uses_full_dataframe`` for ``Rolling*`` and ``Exponential*`` primitives (:pr:`2354`)
+        * Updates for compatibility with upcoming Woodwork release (:pr:`2363`)
```

Comment:
```suggestion
        * Updates for compatibility with upcoming Woodwork release 0.21.0 (:pr:`2363`)
```
This is assuming no breaking change. Might be useful to have version number for future reference